.github/workflows/auto-pr.yml

```
@@ -2,12 +2,39 @@ name: Auto Pull Request
 
 on:
   push:
+    branches:
+      - 'feature/**'
+      - 'fix/**'
+      - 'hotfix/**'
+      - 'docs/**'
     branches-ignore:
```

actionlint (1.7.4)

both "branches" and "branches-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)